Update Turkish translation

bitigchi opened the pull request.

bitigchi committed:
Update Turkish translation

shidel commented:
Actually, could you verify the one is in [fdi-nls](https://github.com/shidel/fd-nls) is up to date.  All of my projects are being restructured and NLS is being removed from the project itself. They are being consolidated and pull directly from the FDI-NLS submodule.

bitigchi commented:
Hello,

I’ve added the FDOS organisation strings and updated the FDIMPLES file to be present in this repository as well.

Best,
Emir

> shidel <notifications@github.com> şunları yazdı (18 May 2020 22:48):
> 
> 
> Actually, could you verify the one is in fdi-nls is up to date. All of my projects are being restructured and NLS is being removed from the project itself. They are being consolidated and pull directly from the FDI-NLS submodule.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
>
